CHANGELOG.md

```diff
@@ -1,73 +1,48 @@
-## 6.0.0-beta.3
```

Comment:
Not sure if we should keep these versions in the CHANGELOG... I guess I'm ok to remove it. 

Also, I see we didn't merge the `rc.1` changes in. So we should make sure there is nothing left from there.

Reply:
Ohhh, shoot! Good catch 🤦‍♂️ Will fix will fix

Reply:
Fixed!